client/components/content-elements/tce-image/edit/index.vue

```diff
@@ -82,23 +81,16 @@ export default {
     isDisabled: { type: Boolean, default: false },
     dense: { type: Boolean, default: false }
   },
-  data: () => ({
-    containerWidth: 0,
-    currentImage: null,
-    persistedImage: null,
-    showCropper: false
-  }),
+  data: () => ({ currentImage: null, persistedImage: null, showCropper: false }),
   computed: {
     showPlaceholder() {
       const imageAvailable = !isEmpty(this.element.data.url);
       if (imageAvailable) return false;
       if (this.$refs.cropper) this.$refs.cropper.destroy();
       return true;
     },
-    elementWidth: ({ containerWidth, element }) => element.data.meta?.width,
-    elementHeight: ({ containerWidth, element }) => element.data.meta?.height,
-    maxWidth: ({ containerWidth, elementWidth }) =>
-      containerWidth > elementWidth ? elementWidth : containerWidth,
+    elementWidth: ({ element }) => element.data.meta?.width,
+    elementHeight: ({ element }) => element.data.meta?.height,
```

Comment:
Do we need to explicitly provide the aspect-ratio to `v-img` based on image's width and height since this is already done in `v-img` by default? (cc: @hrvojevu)